docs/GettingStarted.md

```diff
@@ -2466,3 +2469,24 @@ However, additional instrumentation provided by Datadog can be activated alongsi
 | `OpenTracing::FORMAT_TEXT_MAP` | Yes        |                        |
 | `OpenTracing::FORMAT_RACK`     | Yes        | Because of the loss of resolution in the Rack format, please note that baggage items with names containing either upper case characters or `-` will be converted to lower case and `_` in a round-trip respectively. We recommend avoiding these characters or accommodating accordingly on the receiving end. |
 | `OpenTracing::FORMAT_BINARY`   | No         |                        |
+
+## Common issues
+
+### Payload too large
+
+If you suspect that not all traces from your application are being captured, this might be due to traces that are too large to be processed.
+
+If your application creates long running traces, measured in minutes, or traces with a thousands of operations, such traces can go over our processing limit.
+You can enable [debug mode](#tracer-settings) and check if messages similar to `"Dropping trace. Payload too large"` are being emitted. Keep in mind that debug mode is very verbose. You can also inspect the [Datadog agent logs](https://docs.datadoghq.com/agent/guide/agent-log-files/) for similar messages.
```

Comment:
I'm curious if we could do better here -- would it be possible for us to still send the top-level trace to the agent/back-end, with some metadata element indicating that there's missing data?

Reply:
There's an internal work-in-progress RFC documenting how to tag spans as effective root or not. There's some special logic needed around its implementation, but it's something that we are working on.